tools/GenerateModules.ps1

```diff
@@ -73,6 +73,12 @@ $AllowPreRelease = $true
 if ($ModulePreviewNumber -eq -1) {
     $AllowPreRelease = $false
 }
+
+# Build AutoREST.PowerShell submodule.
+Set-Location (Join-Path $ScriptRoot "../autorest.powershell")
```

Comment:
Will this change the working directory?

Reply:
It does since rush rebuild/build has no path parameter. This is not an issue, though, since the generate modules script is working directory agnostic, we always use `$ScriptRoot` when referencing paths.